validator/src/main.rs

```diff
@@ -1331,6 +1334,27 @@ pub fn main() {
                     "maximum total uncompressed file size of downloaded genesis archive",
                 ),
         )
+        .arg(
+            Arg::with_name("tpu_sigverify_batch_size")
+                .long("tpu-sigverify-batch-size")
+                .value_name("NUMBER")
+                .takes_value(true)
+                .default_value(default_sigverify_batch_size)
+                .hidden(true)
+                .help(
+                    "Number of packets to sigverify at a time",
+                ),
+        )
+        .arg(
+            Arg::with_name("tpu_packet_rate_per_second")
+                .long("tpu-packet-rate-per-second")
+                .value_name("NUMBER")
+                .takes_value(true)
+                .hidden(true)
+                .help(
+                    "Limit the TPU throughput by this many packets per second.",
+                ),
+        )
```

Comment:
If we're going to take this route, can we add some guard rails so that a naive operator can't promote dangerous values to others?

Reply:
wdyt: https://github.com/solana-labs/solana/pull/22614/files#diff-59e2068dd4d49c21ab87f3a5d8792d2daca826ab6924b32413e551c0e3035f61R1935

Reply:
Thanks! Values look good.  Can we use a clap validator instead though?